sfcasts/ep1/data-providers.md

```diff
@@ -1,42 +1,42 @@
 # Data Providers
 
-We treat our source code as first-class citizens. Why not do the same with our
+We treat our source code as a first-class citizen. That means, among other things,
+we avoid duplication. Why not do the same with our
 tests? Our three tests for the size are... repetitive. They test the same thing
-just with slightly different input and then a different assertion. We can improve
-this by using PHPUnit's Data Providers - which run the same test over and over
-with different sets of data.
+just with *slightly* different input and then a different assertion. Is there
+a way to improve this? Absolutely: thanks to PHPUnit Data Providers.
 
 ## Refactor our tests 
 
-Move down to the bottom of our `DinosaurTest` and add
+Move to the bottom of `DinosaurTest` and add
 `public function sizeDescriptionProvider()`. Inside, `yield` an array with `[10, 'Large']`,
 then `yield [5, 'Medium']`, and finally `yield [4, 'Small']`. Yield is just a fancy
-way of returning arrays using PHP's built-in Generator function.
+way of returning arrays using PHP's built-in Generator function. As you'll see in
+a minute, these keys - like `10` and `large` will become *arguments* to our test.
 
-Alrighty, up in our test method, add the `int $length` argument and then
-`string $expectedSize`. Now instead of Big Eaty's length being `10`, we'll use
-`$length`. For our assertion, use `$expectedSize` instead of `Large`. We do not
-need the medium and small tests anymore, so remove both of those methods too.
+Alrighty, up in our test method, add an `int $length` argument and then
```

Comment:
`the` -> `an`. Using `the` makes it sounds like the user understand that, due to something we just did, of course we now need to go add THE `$length` argument. 

But since we're teaching this concept, the user doesn't really understand (yet) the connection between the data provider and the arguments. So, we're showing them how we add `an int $length` argument... and soon they'll see why we did that.

Not sure if that makes any sense :)